policy/modules/system/miscfiles.if

```diff
@@ -82,6 +82,7 @@ interface(`miscfiles_manage_all_certs',`
 	')
 
 	allow $1 cert_type:dir list_dir_perms;
```

Comment:
This is not strictly relevant, but while you're touching this code, you could also remove this line, which is a subset of the interface calls below (it was so already before this commit).